cmd/go.mod

```diff
@@ -22,3 +22,5 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/urfave/cli/v2 v2.3.0
 )
+
+replace github.com/filecoin-project/index-provider => ../
```

Comment:
i don't think you want this, right?

Reply:
I think we do - makes maintenance on our side easier. The only reason we would want to avoid a replace directive is if this module was a library, in which case we'd want to make sure that our users don't need to copy-paste the same replace directives.

Reply:
The issue I see around the corner is having to roll out big changes in cmd module when root module changes are piled up. Removing from this PR since it's a separate issue and beyond the scope of this PR so that we can roll out the changes needed by ignite

Reply:
ah saw @mvdan's message after my comment adding it back in

Reply:
I do not think we want this.  Instead, how about:
https://github.com/filecoin-project/index-provider/pull/97

Reply:
OK I am going to remove replace directive from this PR so that we can push ahead with dependency downgrade and avoid blocking ignite. In another PR we can then continue the discussion re `replace` directive.

Reply:
The discussion to be continued elsewhere probably in #97. Resolving conversation.